transport/conn.go

```diff
@@ -412,8 +412,8 @@ func (c *Conn) sendRequest(req frame.Request, compress, tracing bool) (frame.Res
 	return resp.Response, resp.Err
 }
 
-func (c *Conn) Waiting() int {
-	return int(c.metrics.InQueue.Load() + c.metrics.InFlight.Load())
+func (c *Conn) Waiting() uint32 {
+	return c.metrics.InQueue.Load() + c.metrics.InFlight.Load()
```

Comment:
Why make API ugly?

Reply:
Does this make API ugly? ;o
During studies we were taught that greater specification of types lead to more understandable code,
thanks for the tip!